### ISSUE TITLE
Recursive retrieval of nested items

### ISSUE THREAD

**Issue by szymanskir** [szymanskir/hackeRnews#13]
Some of the retrieved items contain id references to other items. It would be useful to provide an option for the users to retrieve those items as well e.g.

this is the current result
```
List of 7
 $ by    : chr "spez"
 $ id    : int 30
 $ kids  : int [1:2] 31 454446
 $ parent: int 29
 $ text  : chr "Stay tuned..."
 $ time  : POSIXct[1:1], format: "2006-10-10 17:34:59"
 $ type  : chr "comment"
 - attr(*, "class")= chr "hn_item"
```

When using the recursive retrieval we would get the following:
```
List of 7
 $ by    : chr "spez"
 $ id    : int 30
 $ kids  :List of 2
  ..$ :List of 6
  .. ..$ by    : chr "pg"
  .. ..$ id    : int 31
  .. ..$ parent: int 30
  .. ..$ text  : chr "I'm tuned..."
  .. ..$ time  : POSIXct[1:1], format: "2006-10-10 17:40:05"
  .. ..$ type  : chr "comment"
  .. ..- attr(*, "class")= chr "hn_item"
  ..$ :List of 7
  .. ..$ by    : chr "Arrington"
  .. ..$ dead  : logi TRUE
  .. ..$ id    : int 454446
  .. ..$ parent: int 30
  .. ..$ text  : chr "This is why we can't have nice things."
  .. ..$ time  : POSIXct[1:1], format: "2009-01-28 21:37:38"
  .. ..$ type  : chr "comment"
  .. ..- attr(*, "class")= chr "hn_item"
 $ parent: int 29
 $ text  : chr "Stay tuned..."
 $ time  : POSIXct[1:1], format: "2006-10-10 17:34:59"
 $ type  : chr "comment"
 - attr(*, "class")= chr "hn_item"
```

**Comment by szymanskir** [comment]
Actually as it turns out that kids are always of type `comment` than I suggest creating a function `get_comments` that will return a data frame. We do not lose the relation between comments as the `parent` field is available.

**Comment by szymanskir** [comment]
Implemented in #19
